src/stylesheets/components/Products/GridProducts.sass

```
@@ -1,5 +1,5 @@
 .grid
-  display: grid
-  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr))
-  grid-gap: $marg-3
-  width: 100%
+    display: grid
+    grid-template-columns: repeat(auto-fit, minmax(250px, 1fr))
+    grid-gap: $marg-3
```

```suggestion
    gap: $marg-3
```
`grid-gap` is the old syntax